lib/tasks/divisions.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+namespace :divisions do
+  desc 'Migrate from string division to join table divisions'
+  task migrate: :environment do
+    User.all.each do |u|
+      division = Division.where(name: u.division).first_or_create
+      u.divisions << division
+      u.save!
+    end
+  end
```

Comment:
After this task in run in production, I'll run a migration to remove the now-disused string column.